packages/ra-ui-materialui/src/input/RadioButtonGroupInput.spec.js

```diff
@@ -1,123 +1,157 @@
 import React from 'react';
 import expect from 'expect';
-import { render, cleanup } from '@testing-library/react';
-
-import { RadioButtonGroupInput } from './RadioButtonGroupInput';
+import { render, cleanup, fireEvent } from '@testing-library/react';
+import { Form } from 'react-final-form';
 import { TranslationContext } from 'ra-core';
 
+import RadioButtonGroupInput from './RadioButtonGroupInput';
+
 describe('<RadioButtonGroupInput />', () => {
     const defaultProps = {
-        resource: 'bar',
-        source: 'foo',
-        meta: {},
-        input: {},
-        translate: x => x,
+        resource: 'creditcards',
+        source: 'type',
+        choices: [
+            { id: 'visa', name: 'VISA' },
+            { id: 'mastercard', name: 'Mastercard' },
+        ],
     };
 
     afterEach(cleanup);
 
     it('should render choices as radio inputs', () => {
         const { getByLabelText, queryByText } = render(
-            <RadioButtonGroupInput
-                {...defaultProps}
-                label="hello"
-                choices={[
-                    { id: 'M', name: 'Male' },
-                    { id: 'F', name: 'Female' },
-                ]}
+            <Form
+                onSubmit={jest.fn}
+                render={() => (
+                    <RadioButtonGroupInput
+                        {...defaultProps}
+                        label="Credit card"
+                    />
+                )}
             />
         );
-        expect(queryByText('hello')).not.toBeNull();
-        const input1 = getByLabelText('Male');
+        expect(queryByText('Credit card')).not.toBeNull();
+        const input1 = getByLabelText('VISA');
         expect(input1.type).toBe('radio');
-        expect(input1.name).toBe('foo');
+        expect(input1.name).toBe('type');
         expect(input1.checked).toBeFalsy();
-        const input2 = getByLabelText('Female');
+        const input2 = getByLabelText('Mastercard');
         expect(input2.type).toBe('radio');
-        expect(input2.name).toBe('foo');
+        expect(input2.name).toBe('type');
         expect(input2.checked).toBeFalsy();
     });
 
     it('should use the input parameter value as the initial input value', () => {
```

Comment:
you've completely changed the tes implementation, so the test title no longer corresponds to the implementation.

Reply:
I don't really get that comment. `initialValues` provides the initial input value and we check that is indeed applied to the underlying input

Reply:
we no longer use the 'input parameter value', as the test description says